### ISSUE TITLE
[Infrastructure] Nice message on Entering Full Room

### ISSUE THREAD

**Issue by gk-patel** [freeboardgames/FreeBoardGames.org#727]
Currently, when we enter a full room, the following message is displayed,

![image](https://user-images.githubusercontent.com/42677721/102881442-2347a300-444d-11eb-8acf-209dae7c2c8b.png)

Better would be, either to dispaly "Housefull, go create another room" OR "The Room is full, stick around if you want to spectate the match"

**Comment by flamecoals** [comment]
Agree, at the bare minimum we should change the text for each case.
